code-challenges.test.js

```diff
@@ -37,11 +57,26 @@ var randomNouns1 = ["streetlamp", "potato", "teeth", "conclusion", "nephew"]
 var randomNouns2 = ["temperature", "database", "chopsticks", "mango", "deduction"]
 // Expected output: ["Temperature", "Database", "Chopsticks", "Mango", "Deduction"]
 
-
+//create test cases with describe, it, and expect statements
+describe("capFirst", () => {
+  it("returns an array with all the words capitalized", () => {
+    //set expect to return an array with the first letter capitalized
+    expect(capFirst(randomNouns1)).toEqual(["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew"])
+    expect(capFirst(randomNouns2)).toEqual(["Temperature", "Database", "Chopsticks", "Mango", "Deduction"])
+  })
+})
 
 // b) Create the function that makes the test pass.
 
+//creates function capFirst
+const capFirst = (array) => {
+  //use .map() method to iterate the length of the array
+  //use .toUpperCase() method to capitalize the first letter
+  //use .substring() method to return the remainder of the string
+  return array.map(value => value[0].toUpperCase() + value.substring(1, value.length))
+}
 
+//verified function and test cases passed
```

Comment:
Nice job! You definitely have working code. Just a small refactor could be removing the second argument in the `substring` method since it will default to the length.